metaBuild.sh

```diff
@@ -18,7 +18,7 @@ if [ "$1" = "build" ]; then
     CMAKE_FLAGS+=" -DCMAKE_BUILD_TYPE=Debug"
   fi 
 
-  if [ "$SYSTEM_NAME" = "Darwin" ]; then
+  if [ "$SYSTEM_NAME" = "Darwin" -o "$SYSTEM_NAME" = "Linux" ]; then
     echo "Building for OSX"
```

Comment:
Would be good to change this echo string - I could imagine it being confusing to see "Building for OSX" when building on Linux.